### DISCUSSION TITLE
Anyline Preproc

### DISCUSSION THREAD

**Discussion by inferno46n2** [TheMistoAI/MistoLine#8] [Announcements]
Hello,

I may be blind - but is the preprocessor available as well?

Fantastic work - by far the best lineart for XL and was much needed.

**Comment by xyhStruggler** [comment]
We will publish it next week. Stay tuned!
Before that, you can use any other line pre-processor with MistoLine.

**Comment by alenhrp** [comment]
Very strong work, thanks~!
But is there any way to extract a more adapted line drawing? Suggestions would be appreciated, thanks!

**Comment by xyhStruggler** [comment]
> Very strong work, thanks~! But is there any way to extract a more adapted line drawing? Suggestions would be appreciated, thanks!

We will publish a new lineart pre-processor (for ComfyUI) next week which works better with MistoLine.

**Comment by alenhrp** [comment]
> > Very strong work, thanks~! But is there any way to extract a more adapted line drawing? Suggestions would be appreciated, thanks!
> 
> We will publish a new lineart pre-processor (for ComfyUI) next week which works better with MistoLine.

Very helpful work, looking forward to it!

**Comment by 602387193c** [comment]
等你们的预处理出来喽！我做个视频给你们宣传下！如果你们都是中国人，干嘛要说英语呢。

**Comment by inferno46n2** [comment]
> 等你们的预处理出来喽！我做个视频给你们宣传下！如果你们都是中国人，干嘛要说英语呢。

我的意思是，我们都在手机上翻译？

**Comment by xyhStruggler** [comment]
> 等你们的预处理出来喽！我做个视频给你们宣传下！如果你们都是中国人，干嘛要说英语呢。

谢谢！敬请期待~

**Comment by IH8Antelope** [comment]
Would love to see some example workflows :)

**Comment by xyhStruggler** [comment]
> Would love to see some example workflows :)

[CN_mistoline.json](https://github.com/TheMistoAI/MistoLine/files/15293603/CN_mistoline.json)
Here is a simple ComfyUI example that you can try.

**Comment by xyhStruggler** [comment]
I will transfer this issue as a discussion. See you there!

**Comment by xyhStruggler** [comment]
Anyline is here! Thanks for waiting!
https://github.com/TheMistoAI/ComfyUI-Anyline
